sdk/openai/azure-ai-openai-assistants/src/main/java/com/azure/ai/openai/assistants/implementation/Util.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai.assistants.implementation;
+
+import com.azure.ai.openai.assistants.AssistantsServiceVersion;
+import com.azure.core.http.rest.RequestOptions;
+
+/**
+ * Utility class to be used by the SDK internally.
+ */
+public class Util {
```

Comment:
Call this type `AzureOpenAIUtils` or `OpenAIUtils`. `Util` is very generic and hard to search in IDE.